linkerd/app/inbound/src/policy/mod.rs

```diff
@@ -86,20 +72,20 @@ impl AllowPolicy {
         &self,
         client_addr: Remote<ClientAddr>,
         tls: tls::ConditionalServerTls,
-    ) -> Result<Permitted, DeniedUnauthorized> {
+    ) -> Result<Permit, DeniedUnauthorized> {
```

Comment:
any particular rationale behind the name change for `Permit`? w/ the previous naming scheme, the two sides of the `Result` had consistent tense... not a big deal, just wondering.

Reply:
the target types hold "a permit"